src/actions/AsyncActionType.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
Can you remove the related actions from frontend-auth?

Reply:
👍